subsys/usb/usb_device.c

```diff
@@ -179,9 +180,9 @@ static void usb_print_setup(struct usb_setup_packet *setup)
 	LOG_DBG("Setup: %x %x %x %x %x",
 		setup->bmRequestType,
 		setup->bRequest,
```

Comment:
`bRequest` and `bmRequestType` are already in host-endianess?

Reply:
```
/** setup packet definitions */
struct usb_setup_packet {
	u8_t bmRequestType;  /**< characteristics of the specific request */
	u8_t bRequest;       /**< specific request */
	u16_t wValue;        /**< request specific parameter */
	u16_t wIndex;        /**< request specific parameter */
	u16_t wLength;       /**< length of data transferred in data phase */
};
```